lib/check_statistics.rb

```diff
@@ -95,7 +95,7 @@ def number_of_whatsapp_conversations(team_id, start_date, end_date, type = 'all'
             all = 0
             user = 0
             business = 0
-            data['conversation_analytics']['data'][0]['data_points'].each do |data_point|
+            data.dig('conversation_analytics', 'data', 0, 'data_points').each do |data_point|
```

Comment:
@melsawy, if we make this change, what is going to trigger this line? -> https://github.com/meedan/check-api/blob/develop/lib/check_statistics.rb#L115. My suggestion would be to raise an exception if that array is empty but the workspace has `TiplineMessage`'s for the month.

Reply:
Yes, better to keep it as is, then.